### PR TITLE
feat (#major); uniswap-forks; add uniswap forks skinny subgraphs

### PR DESCRIPTION
**Pull request by dhruv-chauhan**
Deployments,

- [uniswap-v2-skinny-ethereum](https://okgraph.xyz/?q=dhruv-chauhan%2Funiswap-v2-skinny)
- [quickswap-skinny-polygon](https://okgraph.xyz/?q=dhruv-chauhan%2Fquickswap-skinny)
- [pancakeswap-v3-skinny-bsc](https://okgraph.xyz/?q=dhruv-chauhan%2Fpancakeswap-v3-skinny)

Validation query,
- uniswap-v2: [query](https://api.thegraph.com/subgraphs/id/QmbAreB7zPavfTrKQawXJLFqytcqq9bC5WxqBC2b6L5JLR/graphql?query=%7B%0A++swaps%28orderBy%3A+timestamp%2C+orderDirection%3A+asc%29+%7B%0A++++id%0A++++hash%0A++++logIndex%0A++++protocol%7Bid%7D%0A++++to%0A++++from%0A++++blockNumber%0A++++timestamp%0A++++tokenIn%7Bid%7D%0A++++amountIn%0A++++amountInUSD%0A++++tokenOut%7Bid%7D%0A++++amountOut%0A++++amountOutUSD%0A++++%23+reserveAmounts%0A++++pool%7Bid%7D%0A++%7D%0A%7D)
- uniswap-v2-skinny: [query](https://api.thegraph.com/subgraphs/id/Qme1iKQ8GW7jJbeHXoAszpCwGdebCvXfn5G6eDko7iDvwf/graphql?query=%7B%0A++swaps%28orderBy%3A+timestamp%2C+orderDirection%3A+asc%29+%7B%0A++++id%0A++++hash%0A++++logIndex%0A++++protocol%7Bid%7D%0A++++to%0A++++from%0A++++blockNumber%0A++++timestamp%0A++++tokenIn%7Bid%7D%0A++++amountIn%0A++++amountInUSD%0A++++tokenOut%7Bid%7D%0A++++amountOut%0A++++amountOutUSD%0A++++reserveAmounts%0A++++pool%7Bid%7D%0A++%7D%0A%7D)

### CODE REVIEW

File: subgraphs/uniswap-forks-skinny/protocols/uniswap-v2-skinny/config/templates/uniswap.v2.skinny.template.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        - name: Pair
+          file: ./abis/uniswap-v2/Pair.json
+        - name: Factory
+          file: ./abis/uniswap-v2/Factory.json
+        - name: TokenABI
+          file: ./abis/uniswap-v2/ERC20.json
+      eventHandlers:
+        - event: Swap(indexed address,uint256,uint256,uint256,uint256,indexed address)
+          handler: handleSwap
+        - event: Sync(uint112,uint112)
```
Comment (dmelotik) [inline subgraphs/uniswap-forks-skinny/protocols/uniswap-v2-skinny/config/templates/uniswap.v2.skinny.template.yaml:81]: Can you remove the sync event, we don't need it for anything afaik

**Comment by dmelotik** [comment]
Nice work @dhruv-chauhan ! In order to get it syncing now I am gonna remove the `sync` event locally and deploy to the messari hosted service. Once you remove it here, I will merge this PR.

Good idea making another folder for the skinny subgraphs

**Comment by dhruv-chauhan** [comment]
@dmelotik I intentionally kept the 'Sync' event. It gives us the updated pool's token balances, which helps us in setting 'swap.reserveAmounts'.

(If we do have to remove it, ig we'll have to do the contract call on pool)

**Comment by dmelotik** [comment]
@dhruv-chauhan we should be good without it, since we simply only need `swap.amountIn/Out` and `swap.timestamp` and `swap.tokenIn/Out`

File: deployment/deployment.json [hunk 2]
```diff
@@ -3855,6 +3855,58 @@
       }
     }
   },
+  "pancakeswap-v3-swaps": {
```
Comment (dmelotik) [inline deployment/deployment.json:3858]: I am sorry if I confused you, I already deployed with the word `swap` and not `swaps`...

Comment (dhruv-chauhan) [reply deployment/deployment.json]: my bad, corrected.

**Review by dmelotik — APPROVED** [review]
lgtm

**Pull request merged**